### PR TITLE
168259456 upadate monitor group module

### PR DESCRIPTION
**Pull request by giskarda**

### CODE REVIEW

File: azurerm_monitor_action_group/output.tf [hunk 1]
```diff
@@ -1,3 +1,3 @@
 output "name" {
-  value = "${var.azurerm_monitor_action_group_name}"
+  value = "${var.azurerm_monitor_action_group_name_suffix}"
```
Comment (LucaPrete) [inline azurerm_monitor_action_group/output.tf]: This is wrong:
* The output name doesn't respect conventions (should be azurerm_monitor_action_group_name)
* The value is also wrong. The be the local one, not the suffix
* Description missing

File: azurerm_monitor_action_group/variables.tf [hunk 2]
```diff
@@ -1,7 +1,18 @@
...
+  description = "The prefix used to name all resources created."
+}
+
+# Specific variables for monitor action group
+
+variable "azurerm_monitor_action_group_name_suffix" {
+  description = "Suffix for the monitor action group name."
+
 variable "azurerm_monitor_action_group_short_name" {
   description = "Short name (1-12 characters) for the monitot group"
```
Comment (LucaPrete) [inline azurerm_monitor_action_group/variables.tf]: typo

File: azurerm_monitor_action_group/variables.tf [hunk 3]
```diff
@@ -1,7 +1,18 @@
...
+variable "resource_name_prefix" {
+  description = "The prefix used to name all resources created."
+}
+
+# Specific variables for monitor action group
+
+variable "azurerm_monitor_action_group_name_suffix" {
+  description = "Suffix for the monitor action group name."
+
 variable "azurerm_monitor_action_group_short_name" {
```
Comment (LucaPrete) [inline azurerm_monitor_action_group/variables.tf]: Not sure what's this for...

File: azurerm_monitor_action_group/variables.tf (hunk 2 above)
Comment (giskarda) [reply azurerm_monitor_action_group/variables.tf]: Where is the typo?

File: azurerm_monitor_action_group/variables.tf (hunk 3 above)
Comment (giskarda) [reply azurerm_monitor_action_group/variables.tf]: This will generate azurerm_monitor_action_group_name

**Pull request merged**